.azure-pipelines/net472.yml

```diff
@@ -0,0 +1,36 @@
+variables:
+  WindowsName: windows
+  WindowsImage: VS2017-Win2016
+  TestFramework: net472
+  TestTarget: TestNet472
+  Configuration: Debug
```

Comment:
If this is a configuration you want to enable at queue time then you will need to remove it from here and put it only in the build pipeline variables tab.

Reply:
This is not intended to be build-time variables. I use these simply as global variables to all jobs in the yaml. If there was a concept of hard-coded values (constants), I would have used them here. Also, it is convenient to have all values that may change in the future listed at the top of the file.